tikv/deploy-tikv-docker-compose.md

```diff
@@ -44,7 +44,7 @@ Make sure you have installed the following items on your machine:
 
     ```bash
     cd tidb-docker-compose/compose
-    networkMode: host
+    sed -i "s/^networkMode: bridge/networkMode: host/g" values.yaml
```

Comment:
This only modifies `networkMode`, but this step we should modify `networkMode` and comment the TiDB section out. So perhaps using following command would be better:
```
cd tidb-docker-compose
vim compose/vlaues.yaml
```
The description in this step is already clear to users how to modify values.yaml using vim.